commands/src/main/java/org/jclouds/karaf/commands/table/internal/OSGiScriptEngine.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
> No changes to the notice file should be required as the copyright owner of the copied files is already the ASF.

Ah, good to know. Then I think we should be OK just to reformat and perhaps tweak the style where applicable?